src/game/Game.ts

```diff
@@ -0,0 +1,128 @@
+import KeyListener from "./KeyListener";
+
+enum Tile {
+    FLOOR,
+    WALL,
+    UNDEFINED,
+}
```

Comment:
Not sure if you've seen this commit (ff6490860fac2a59247597c14ffe79d6da7983e0) yet, but as an extension to this `enum Tile`, we might want to consider classes for tiles.